### ISSUE TITLE
documentation in wiki

### ISSUE THREAD

**Issue by jhnnsnk** [HBPVIS/VIOLA#9]
- [x] only user manual?
- [ ] needs review, more details

For Developer Manual, the following points are missing:
- [ ]  How to develop your own visualization and add it to VIOLA
- [ ]  How to develop or integrate your own data loading routine

**Comment by jhnnsnk** [comment]
- [ ] only user manual?
- [ ] needs review, more details

**Comment by bweyers** [comment]
I added a first version of the developer manual. It is questionable if the wiki is a right place but for the first attempt I think that it is feasible. Nevertheless, it has to be revised and extended.

**Comment by espenhgn** [comment]
The whole wiki setup appears now a bit odd. Shouldn't 'Documentation' be what is on the wiki front page, linking directly to the user manual and developer manual?

**Comment by bweyers** [comment]
That is true! I fixed it and added the content of the documentation page to the home page. I deleted the old documentation page.
